README.md

```
@@ -261,3 +261,30 @@ Reference designs for the project can be found here:
 **Note:** This project is part of **GSoC'25 under AOSSIE** and is actively under development.
 
 
+/**
+## ✨ README Improvement Notes
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)